lib/BadgeOverlaySupport/BadgeOverlaySupport.less

```diff
@@ -0,0 +1,25 @@
+.moon-badge-overlay-support {
+	.moon-image-client {
```

Comment:
Is it possible to create the same effect without going into the internal classes of .moon-image? Doing so creates a more fragile and interdependent system.